active/InformationBar/InformationBar.md

```diff
@@ -0,0 +1,512 @@
+
+A custom icon can be set via the IconSource property. By default, an icon will be visible. This icon can be removed by setting the IsIconVisible property to false. For custom icons, the recommended icon size is 20px.
+
+XAML
+```xml
+<StackPanel x:Name="ContentArea">
+    <controls:InfoBar x:Name="NewPhotosNotification"
+        Title="New Photos From Cloud"
+        Message="Sync your latest photos to the Cloud"
+        Background="#FF557EB9"
+        IconSource="Camera">
+        <controls:InfoBar.ActionButton>
+            <Button Content="Upload" Command="{x:Bind UploadPhoto}"/>
+        </controls:InfoBar.ActionButton>
+    </controls:InfoBar>
+</StackPanel>
+```
+
+![A mockup of an InfoBar in the top of the content area. The notification's title is "New Photos From Cloud" and it's message is "Sync your latest photos to the Cloud". It has a bright green accent color and a "Camera" icon.](images/Custom_IconColor.png)
+
+## Add an action button
+By default, an 'X' close button will appear as the right most component in the bar.
+
+An additional action button can be added by defining your own button that inherits [ButtonBase](https://docs.microsoft.com/en-us/uwp/api/Windows.UI.Xaml.Controls.Primitives.ButtonBase) and setting it in the ActionButton property. Custom styling will be applied to action buttons of type Button and HyperlinkButton for consistency and accessibility. Outside of the ActionButton, additional action buttons can be added via custom content and will appear below the message.
+
+XAML
+```xml
+<StackPanel x:Name="ContentArea">
+    <controls:InfoBar x:Name="UnsuccessfulSaveNotification"
+        Severity="Warning"
+        Title="Error while saving"
+        Message="Your document was unable to be saved.">
+        <controls:InfoBar.ActionButton>
+            <Button Content="Try Again" Command="{x:Bind SaveDocuments}"/>
+        </controls:InfoBar.ActionButton>
+    </controls:InfoBar>
+</StackPanel>
+```
+
+![A mockup of an InfoBar with a single line message and an action button](images/Warning_Button.png)
+
+XAML
+```xml
+<StackPanel x:Name="ContentArea">
+    <controls:InfoBar x:Name="ConnectionErrorNotification"
+        Title="Error while saving"
+        Message="Lorem ipsum long message">
+        <controls:InfoBar.ActionButton>
+            <HyperlinkButton
+                Content="www.microsoft.com" 
+                NavigateUri="http://www.microsoft.com"/>
+        <controls:InfoBar.ActionButton/>
+    </controls:InfoBar>
+</StackPanel>
+```
+![A mockup of an InfoBar with a single line message and hyperlink](images/Default_MessageHyperlink.png)
+![A mockup of an InfoBar with a message expanding multiple lines and a hyperlink](images/Docked_MultiLineIconTitleHyperlink.png)
+
+## Custom content
+Content can be added to an InfoBar using the Content property. It will appear below the message and above the action or hyperlink buttons if defined and multiline. The InfoBar will expand to fit the content defined.
```

Comment:
What does it mean to be multiline?

(Maybe this is answered in the next example and they just need to re-ordered)